app/Datagrids/Filters/CharacterFilter.php

```diff
@@ -75,6 +75,7 @@ public function build()
             ->hasAttributes()
             ->tags()
             ->attributes()
+            ->connection()
```

Comment:
All the other filters are plural, so would be nice to keep the same nomenclature :)
Also needs to be added to Locations, Families, Tags, Calendars, etc